### PR TITLE
docs: clarify usage of host based authentication

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: blackbox/docs/admin/auth/hba.rst [hunk 1]
```diff
@@ -19,9 +19,20 @@ See: :ref:`applying-cluster-settings`.
 
 .. NOTE::
 
+   The ``crate.yml`` that is shipped with CrateDB explicitly enables host based
+   authentication and defines a set of sane rules, which take affect in case
```
Comment (seut) [inline blackbox/docs/admin/auth/hba.rst]: should phrase ".. take effect" afaik

File: blackbox/docs/admin/auth/hba.rst [hunk 2]
```diff
@@ -19,9 +19,20 @@ See: :ref:`applying-cluster-settings`.
...
+   HBA and Enterprise features are enabled.
+
    Host Based Authentication is an
    :ref:`enterprise feature <enterprise_features>`.
 
+.. TIP::
+
+   If you want to use Enterprise features, but do not require authentication,
+   you need to set ``auth.host_based.enabled`` to ``false``. However,
+   additionally you still need to provide the user ``crate`` when connecting to
```
Comment (seut) [inline blackbox/docs/admin/auth/hba.rst]: afaik this is not true when using HTTP protocol, maybe we should mention?

Comment (chaudum) [reply blackbox/docs/admin/auth/hba.rst]: Pushed a fixup which I hope clarifies not required user when using http.

Comment (seut) [reply blackbox/docs/admin/auth/hba.rst]: lgtm, thx.

**Comment by chaudum** [comment]
re-test this please

**Pull request merged**